config.template.toml

```diff
@@ -79,8 +79,12 @@ persist_sandbox = false
 # Use host network
 #use_host_network = false
 
+# Name of the default agent
+#default_agent = "CodeActAgent"
+
 #################################### LLM #####################################
-# Configuration for the LLM model
+# Configuration for LLM models (group name ends with 'llm')
```

Comment:
Does it end with 'llm'?

Reply:
Ah thx, I went back and forth on design decisions and this was a leftover I forgot to clean up